services/utils/classNames.js

```diff
@@ -0,0 +1,3 @@
+export function classNames(...classes) {
+    return classes.filter(Boolean).join(" ");
+}
```

Comment:
Missing ending blank line

```suggestion
}

```